tests/js/common/http/grants-spec.js

```diff
@@ -134,3 +135,216 @@ describe('Grants', function() {
     expect(JSON.parse(resp.body)).to.have.property('result', 'rw');
   });
 });
+
+describe('UserProperties', function () {
+  const rootUser = 'root';
+  const rootDB = '_system';
+
+  const nonRootDB = 'nonRootDB';
+  const nonRootUser = 'notRoot';
+
+  const configPropertyAttribute = 'graphs';
+  const additionalPropertyAttribute = 'someAdditionalAttribute';
+
+  const initConfigObject = {
+    a: true,
+    b: false,
+    c: 1,
+    d: 1.1,
+    e: "aString",
+    f: [1, 2, 3],
+    g: {"anObjectKey": "anObjectValue"}
+  };
+
+  const overwriteConfigObject = {
+    c: 1337,
+    someNewParam: "MJ7"
+  };
+
+  const mergeConfigObject = {
+    "thisMustStay": true
+  };
+
+  const verifyEmptyConfig = (result) => {
+    console.warn(result);
```

Comment:
Ideally tests shouldn't log requests/responses unless they are flaky.
```suggestion
```

Reply:
Good! jslint did not complain :(